docs/reference/docs/bulk.asciidoc

```diff
@@ -138,7 +136,6 @@ The result of this bulk operation is:
                "successful": 1,
                "failed": 0
             },
-            "created": true,
```

Comment:
This is exactly the kind of thing I'd miss when removing these without the assertions.